src/test/ui/consts/const-size_of-cycle.stderr

```diff
@@ -1,17 +1,27 @@
-error[E0391]: cycle detected when computing layout of `Foo`
+error[E0391]: cycle detected when const-evaluating + checking `Foo::bytes::{{constant}}`
+  --> $DIR/const-size_of-cycle.rs:16:17
+   |
+LL |     bytes: [u8; std::mem::size_of::<Foo>()]
+   |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
-note: ...which requires normalizing `ParamEnvAnd { param_env: ParamEnv { caller_bounds: [], reveal: All }, value: [u8; _] }`...
 note: ...which requires const-evaluating `Foo::bytes::{{constant}}`...
   --> $SRC_DIR/libcore/mem.rs:LL:COL
    |
 LL |     unsafe { intrinsics::size_of::<T>() }
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^
-   = note: ...which again requires computing layout of `Foo`, completing the cycle
-note: cycle used when const-evaluating `Foo::bytes::{{constant}}`
-  --> $SRC_DIR/libcore/mem.rs:LL:COL
+note: ...which requires computing layout of `Foo`...
+note: ...which requires normalizing `ParamEnvAnd { param_env: ParamEnv { caller_bounds: [], reveal: All }, value: [u8; _] }`...
+note: ...which requires const-evaluating + checking `Foo::bytes::{{constant}}`...
+  --> $DIR/const-size_of-cycle.rs:16:17
    |
-LL |     unsafe { intrinsics::size_of::<T>() }
-   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^
+LL |     bytes: [u8; std::mem::size_of::<Foo>()]
+   |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
+   = note: ...which again requires const-evaluating + checking `Foo::bytes::{{constant}}`, completing the cycle
+note: cycle used when processing `Foo`
```

Comment:
Is it expected that "requires const-evaluating + checking `Foo::bytes::{{constant}}`" occurs *twice* at the end of this cycle?

> ...which requires const-evaluating + checking `Foo::bytes::{{constant}}`...
> ...which again requires const-evaluating + checking `Foo::bytes::{{constant}}`, completing the cycle